en-US/snippets/2017/jul2017.lang

```diff
@@ -0,0 +1,145 @@
+# Cinch means very easy.
+;Protecting yourself online can be a lot of work. Luckily, <a>Firefox gives you the tools that make privacy a cinch.</a>
+Protecting yourself online can be a lot of work. Luckily, <a>Firefox gives you the tools that make privacy a cinch.</a>
+
+
+# Desktop Snippet
+# Link: https://support.mozilla.org/products/firefox/protect-your-privacy
+;Always feel like somebody is watching you? Control your data, block malware and outfox the trackers with <a>Firefox’s privacy protection tips.</a>
+Always feel like somebody is watching you? Control your data, block malware and outfox the trackers with <a>Firefox’s privacy protection tips.</a>
+
+
+# Desktop Snippet
+# https://support.mozilla.org/kb/get-started-firefox-overview-main-features
+;<a>Become a power Firefox user.</a> Discover customization tricks and something we call, THE AWESOME BAR.
+<a>Become a power Firefox user.</a> Discover customization tricks and something we call, THE AWESOME BAR.
+
+
+# Desktop Snippet
+# https://support.mozilla.org/kb/get-started-firefox-overview-main-features
+;Whether you’re new to Firefox or just need a refresher, here’s a quick <a>guide to all the awesomeness Firefox comes with</a> right out of the box.
+Whether you’re new to Firefox or just need a refresher, here’s a quick <a>guide to all the awesomeness Firefox comes with</a> right out of the box.
+
+
+# Desktop Snippet
+;<a>Fast. Small. Private.</a>
+<a>Fast. Small. Private.</a>
+
+
+# Desktop Snippet
+You don't need to sacrifice a lot of data to experience the best in private browsing. Send Firefox Focus to your phone.
+;You don't need to sacrifice a lot of data to experience the best in private browsing. Send Firefox Focus to your phone.
+
+
+# Desktop Snippet
+;<a>Meet your new favorite browser.</a>
+<a>Meet your new favorite browser.</a>
+
+
+# Desktop Snippet
+;Small app + major freedom = Firefox Focus, your new favorite private browser. Send a download link to your phone.
+Small app + major freedom = Firefox Focus, your new favorite private browser. Send a download link to your phone.
+
+
+# Desktop Snippet
+# Try to find words that mean small but also rhyme.
+;<a>Strong and mighty. Teeny tiny.</a>
+<a>Strong and mighty. Teeny tiny.</a>
+
+
+# Desktop Snippet
+;Meet Firefox Focus, the super-private browser that’s only 4MB. Send a download link to your phone.
+Meet Firefox Focus, the super-private browser that’s only 4MB. Send a download link to your phone.
+
+
+# Desktop Snippet
+;<a>Tiny browser. Big freedom.</a>
+<a>Tiny browser. Big freedom.</a>
+
+
+# Desktop Snippet
+;Stay hidden from trackers. Save space on your phone. Get Firefox Focus, the super-private browser that’s only 4MB. Send yourself a download link.
+Stay hidden from trackers. Save space on your phone. Get Firefox Focus, the super-private browser that’s only 4MB. Send yourself a download link.
+
+
+# Desktop Snippet
+;<a>Focus: 36 and counting. Ads: 0.</a>
+<a>Focus: 36 and counting. Ads: 0.</a>
```

Comment:
I should have asked, but I think this refers to the time spent on the app but no ad during the entire time.